device/Microsoft.Azure.Devices.Client/Transport/AmqpTransportHandler.cs

```diff
@@ -638,6 +642,25 @@ public override Task RejectAsync(string lockToken, CancellationToken cancellatio
             }
         }
 
+        private void verifyResponseMessage(AmqpMessage response)
```

Comment:
Nit: From the [coding standards ](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/doc/coding-style.md) - Methods must begin with upper-case: `VerifyResponseMessage`.

Reply:
Fixed